internal/pkg/mapper/relations/link.go

```diff
@@ -8,22 +8,7 @@ import (
 	"github.com/keboola/keboola-as-code/internal/pkg/utils"
 )
 
-// MapAfterLocalLoad - load relations from manifest to object.
```

Comment:
Rozdelil som velke subory v `mapper` napr. `load.go`, na mensie -> `local_load.go`, `remote_load.go`.
Niekde to tak bolo a niekde nie.
Tym je to viac prehladnejsie a da sa v tom lepsie orientovat.